opentimelineio/adapters/cmx_3600.py

```diff
@@ -858,6 +893,27 @@ def __init__(
         line.reel = _reel_from_clip(clip)
         line.source_in = clip.source_range.start_time
         line.source_out = clip.source_range.end_time_exclusive()
+
+        timing_effect = _relevant_timing_effect(clip)
+
+        if timing_effect:
+            if timing_effect.effect_name == "FreezeFrame":
```

Comment:
Maybe isinstance would be better than checking by name?

Reply:
In this case, `isinstance` would catch any derived classes as well, and I want `FreezeFrame` effects and _only_ `FreezeFrame` effects.